src/main/java/org/humancellatlas/ingest/project/ProjectQueryBuilder.java

```diff
@@ -28,10 +31,37 @@ public static Query buildProjectsQuery(SearchFilter searchFilter) {
         Criteria queryCriteria = new Criteria().andOperator(criteria_list.toArray(new Criteria[criteria_list.size()]));
         Query query = new Query().addCriteria(queryCriteria);
         Optional.ofNullable(searchFilter.getSearch())
+                .map(search -> ProjectQueryBuilder.formatSearchString(searchFilter))
                 .ifPresent(search -> {
                     query.addCriteria(TextCriteria.forDefaultLanguage().matching(String.valueOf(search)));
                 });
 
         return query;
     }
+
+    private static final Map<SearchType, Function<SearchFilter, String>> x = Map.of(
```

Comment:
could rename `x` to be `formatter`  for readability?

Reply:
I wanted to find a good name and forgot :(

Reply:
i think you picked good names for it!